app/views/reports/regions/_details_footnotes.html.erb

```diff
@@ -91,4 +81,39 @@
       </div>
     </div>
   <% end %>
+  <% if @region.district_region? || @region.state_region? %>
+    <div class="mb-24px">
+      <p class="mb-8px fs-16px fw-bold c-grey-dark">
+        Hypertension patient coverage
+      </p>
+      <div class="pl-16px">
+        <p class="mb-8px fs-14px c-grey-dark">
+          <%= t("hypertension_patient_coverage_copy") %>
+        </p>
+      </div>
+    </div>
+  <% end %>
+  <% if @region.district_region? %>
+    <div class="mb-0px">
+      <p class="mb-8px fs-16px fw-bold c-grey-dark">
+        Total estimated hypertensive population
+      </p>
+      <div class="pl-16px">
+        <p class="mb-8px fs-14px c-grey-dark">
+          <%= t("total_estimated_hypertensive_population.district_copy", region_name: @region.name) %>
+        </p>
+      </div>
+    </div>
+  <% elsif @region.state_region? %>
+    <div class="mb-0px">
+      <p class="mb-8px fs-16px fw-bold c-grey-dark">
+        Total estimated hypertensive population
+      </p>
+      <div class="pl-16px">
+        <p class="mb-8px fs-14px c-grey-dark">
+          <%= t("total_estimated_hypertensive_population.state_copy", region_name: @region.name) %>
+        </p>
+      </div>
```

Comment:
If you refactor the one `translate` call (ie `t`) to a helper, this if/else can go away:

```
<%= total_estimated_hypertensive_population_copy(@region) %>
```

then in your helper:

```
def total_estimated_hypertensive_population_copy(region)
  region_copy = "#{region.type}_copy"
  t("total_estimated_hypertensive_population.#{region_copy}", region_name: region.name
end
```